### ISSUE TITLE
Another issue related to IE treated wood

### ISSUE THREAD

**Issue by malte0811** [KubeJS-Mods/KubeJS#26]
See https://github.com/BluSunrize/ImmersiveEngineering/issues/4277

This is apparently different from #14 (reporter in the IE issue is using the latest 1.16.1 build of KubeJS). The error message is
```
[Render thread/ERROR] [KubeJS Server/]: Parsing error loading recipe immersiveengineering:crafting/treated_wood_horizontal: java.lang.IllegalStateException: Tag forge:creosote used before it was bound
```
This shouldn't ever happen in IE code unless you're reading recipes from JSON on the client. I was going to debug this a bit more, but apparently FG doesn't properly deobfuscate Mixins.

**Comment by LatvianModder** [comment]
Hm. This error usually happens due to script not recipe. All tags should already be loaded when recipes get loaded. Ill download Valhhelsia 3 and check it out

**Comment by MaxNeedsSnacks** [comment]
Able to reproduce on Enigmatica 6 version `0.2.16`. It's why we currently have to reload the server once every time after we start it up

**Comment by MaxNeedsSnacks** [comment]
This is especially frustrating because E6 currently has an issue with productivebees where the game crashes on reload...

**Comment by andrewsf** [comment]
I believe [a fix went into Valhelsia 3.0.18](https://github.com/ValhelsiaTeam/Valhelsia/commit/fba19d260602137120b4254372648748995fe583) for this.
